src/integrationTest/resources/application.properties

```diff
@@ -44,7 +44,7 @@ case.maintenance.amend-petition-draft-refusal.context-path=/casemaintenance/vers
 ###############################################
 #  Core Case Data Variables                   #
 ###############################################
-core_case_data.api.url=${CASE_DATA_STORE_BASEURL:http://ccd-data-store-api-aat.service.core-compute-aat.internal}
+core_case_data.api.url=http://ccd-data-store-api-nfdiv-ccd-definitions-pr-16.service.core-compute-preview.internal
```

Comment:
Do you really need to change this ? in preview template you have already set `CASE_DATA_STORE_BASEURL` so what's existing should work ?

Reply:
True. im not sure what the colon in template literals is for tbh, i assumed its a condition if its null use other one but im not sure. This branch isnt going to be used so i think think it matters much.

Reply:
yes it is a condition if first is not set use the other one by default. ok if it is not going to be used it should be fine thanks